### DISCUSSION TITLE
How do I configure the leader key?

### DISCUSSION THREAD

**Discussion by diminutivesloop** [tgrosinger/leader-hotkeys-obsidian#35] [Q&A] [answered]
Am I missing something? I can't find any command in the Obsidian hotkeys settings or the plugin settings that could be used to set the leader key and there is no default leader key documented.

**Comment by diminutivesloop** [comment, marked as answer]
Actually it looks like I had them configured correctly. I just didn't realize this plugin doesn't currently support vim normal mode.

**Comment by koan09** [comment]
How did you end up configuring your leader key?

**Reply by diminutivesloop** [reply]
Sorry but I don't use this plugin anymore and I don't remember how I had it configured when I was using it.

**Comment by jps-2000** [comment]
Came here looking for ctrl-w ctrl-h and ctrl-w ctrl-l to move between tab groups (panes). Seems this plugin got broken at some point in Obsidian's life. I got to where I wanted with the [vimrc support plugin](https://github.com/esm7/obsidian-vimrc-support) and adding this to the .obsidian.vimrc
```
nmap <C-w><C-h> :obcommand<space>editor:focus-left<CR>                                                                                                                                                                                      
nmap <C-w><C-l> :obcommand<space>editor:focus-right<CR>                                                                                                                                                                                     
nmap <C-w><C-k> :obcommand<space>editor:focus-up<CR>                                                                                                                                                                                        
nmap <C-w><C-j> :obcommand<space>editor:focus-down<CR>     
```
You will need to disable the relevant default obsidian hotkey bindings
